flasksrc/schema.sql

```diff
@@ -85,3 +86,23 @@ CREATE TABLE toynet_survey_types (
   PRIMARY KEY (type_id)
   FOREIGN KEY (type_id) REFERENCES toynet_survey_questions (type_id)
 );
+
+CREATE TABLE toynet_topos (
+  topo_id INTEGER NOT NULL, --primary key
+  topology TEXT NOT NULL, --XML file (starting file for a lesson module)
```

Comment:
The comment is a little imprecise, since it is not a file but purely the XML representation of the topology.